content/zh-CN/guides/management-tools.md

```diff
@@ -13,35 +13,20 @@ title: 管理开源项目的工具
   - [利用现有工具](#利用现有工具)
   - [创建仪表板](#创建仪表板)
 - [基本工具集的要素](#基本工具集的要素)
-  - [Automate processes](#automate-processes)
   - [自动化流程](#自动化流程)
-  - [Manage critical tasks](#manage-critical-tasks)
   - [管理关键任务](#管理关键任务)
-  - [Source code management](#source-code-management)
   - [源代码管理](#源代码管理)
-  - [License compliance](#license-compliance)
   - [许可合规性](#许可合规性)
-- [Tools for managing source code](#tools-for-managing-source-code)
 - [管理源代码的工具](#管理源代码的工具)
-  - [Bug and issue tracking](#bug-and-issue-tracking)
   - [错误和问题跟踪](#错误和问题跟踪)
-  - [Archiving and release management](#archiving-and-release-management)
   - [归档和发布管理](#归档和发布管理)
-- [Tools for tracking project health](#tools-for-tracking-project-health)
 - [项目健康跟踪工具](#项目健康跟踪工具)
-  - [For better code reviews:](#for-better-code-reviews)
   - [为了更好的代码审查：](#为了更好的代码审查)
-  - [For Contributor License Agreements](#for-contributor-license-agreements)
   - [对于贡献者许可协议](#对于贡献者许可协议)
-  - [GitHub Management at Corporate Scale](#github-management-at-corporate-scale)
   - [GitHub 在企业规模上的管理](#github-在企业规模上的管理)
-  - [Project Quality](#project-quality)
   - [项目质量](#项目质量)
-- [Tools for communications and collaboration](#tools-for-communications-and-collaboration)
 - [通信与协作工具](#通信与协作工具)
-- [Tools for corporate-scale GitHub management](#tools-for-corporate-scale-github-management)
-- [企业规模 GitHub 管理工具](#企业规模-github-管理工具)
```

Comment:
Is this zh line (ilne 43) intentional to remove?

Reply:
> Is this zh line (ilne 43) intentional to remove?

We'd better focus on the right side file when review , right ? :smile: